package.json

```diff
@@ -157,7 +157,7 @@
     "mocha-junit-reporter": "^2.0.2",
     "mocha-multi-reporters": "^1.5.1",
     "mock-require": "3.0.3",
-    "mongodb": "^5.4.0",
+    "mongodb": "^6.1.0",
```

Comment:
The minimum supported Node.js version is now v16.20.1 [ref](https://github.com/mongodb/node-mongodb-native/releases#:~:text=and%20dependency%20updates-,Minimum,-Node.js%20version)

Reply:
updated the code to incorporate the node version change https://github.com/instana/nodejs/pull/869/commits/6d9487f0f0a5368fafa57d166a6ce3d0f7fd8222. @kirrg001

Reply:
We are currently testing again v3, v4 and v6.
IMO v3 and v4 need to be iterated as well. So that we are testing against v4, v5, v6

Reply:
> is now v16.20.1

I do not understand why this **specific** version is their minimum Node version.

Reply:
> We are currently testing again v3, v4 and v6. IMO v3 and v4 need to be iterated as well. So that we are testing against v4, v5, v6

Updated the code

Reply:
Tests are broken

Reply:
fixed